injected/src/features/broker-protection/types.js

```diff
@@ -94,7 +94,7 @@ export class AsyncProfileTransform {
      * @param {Record<string, any>} profileParams - the original action params from `action.profile`
      * @return {Promise<Record<string, any>>}
      */
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+     
```

Comment:
Can we delete these newlines? I think we should be checking for unused method though.

Reply:
Ah are we not intending to adopt prettier in the same PR? I think we should tbh.

I'm hoping prettier drops unnecessary whitespace at the start of lines like this.

Reply:
I'm hoping to follow up with the Prettier PR (it will need changes in the config repo too). So I'll leave it up to you whether to merge this now or wait until the second PR.